NinchatSDKSwift/Implementations/Data Source - Delegate/Questionnaire/NINQuestionnaireConversationDataSourceDelegate.swift

```diff
@@ -162,7 +160,7 @@ extension NINQuestionnaireConversationDataSourceDelegate {
         cell.isUserInteractionEnabled = (self.elements[indexPath.section].first?.isShown ?? true) && (indexPath.section == self.sectionCount-1)
         cell.overrideAssets(with: self.delegate)
         // Need to check only for the last item if to enable/disable navigation buttons
-        cell.setSatisfaction(self.requirementsSatisfied && indexPath.section == sectionCount-1, lastItem: indexPath.section == sectionCount-1)
+        cell.setSatisfaction(self.viewModel.requirementsSatisfied && indexPath.section == sectionCount-1, lastItem: indexPath.section == sectionCount-1)
```

Comment:
What exactly does this do?

Reply:
It enables/disables questionnaire navigation buttons according to user inputs.